src/api/app/controllers/webui/user_controller.rb

```diff
@@ -14,6 +14,8 @@ def index
 
   def logout
     logger.info "Logging out: #{session[:login]}"
+    key = "ldap_cache_userpasswd:" + session[:login]
+    Rails.cache.delete(key)
```

Comment:
it is set inside of app/models/user_ldap_strategy.rb self.find_with_ldap

I will adapt the code.